### PR TITLE
Microservice for Estimate Effectiveness

### PR DESCRIPTION
**Pull request by Araviraj8**

### CODE REVIEW

**Review by AshankDsouza — DISMISSED** [review]
Lets wait for work capacity to be merged before merging this

**Review by Araviraj8 — COMMENTED** [review]
Work capacity has been merged.

**Comment by Araviraj8** [comment]
Waiting for dockerization.

**Pull request closed without merging**
